hrui/tests/phpunit/HrUITest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\HookInterface;
+use Civi\Test\TransactionalInterface;
+
+
+class HrUITest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, HookInterface, TransactionalInterface {
+
```

Comment:
I did see any hook function in this class, so I think you can get rid of the HookInterface (don't forget its use statement as well)